pytorch_lightning/metrics/functional/classification.py

```diff
@@ -105,10 +107,10 @@ def stat_scores(
         target: target tensor
         class_index: class to calculate over
         argmax_dim: if pred is a tensor of probabilities, this indicates the
-            axis the argmax transformation will be applied over
+            axis the argmax transformation will be applied over (default: 1)
 
     Return:
-        True Positive, False Positive, True Negative, False Negative
+        True Positive, False Positive, True Negative, False Negative, SUP
```

Comment:
I know what this is but not sure what to write here `SUP`.

Reply:
```suggestion
        True Positive, False Positive, True Negative, False Negative, Support
```